cabal-install/Distribution/Client/BuildReports/Anonymous.hs

```diff
@@ -112,6 +112,7 @@ data InstallOutcome
    | ConfigureFailed
    | BuildFailed
    | TestsFailed
+   | BenchFailed
```

Comment:
SInce you removed the v1-install stuff this is now unused so you should also remove it from this PR

Reply:
Good catch. I've removed this.